docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -1,55 +1,69 @@
+
+1.  Grant the Ingress Operator the cluster-admin role (no longer required in 4.20)
+2.  Enable the feature gate (no longer required to 4.20)
+3.  Create a GatewayClass with the controller name `openshift.io/gateway-controller`
 
 Let’s walk through each of these steps in more detail.
 
-First, grant the Ingress Operator the cluster-admin role. This role is required in order for the Ingress Operator to
-create the SMCP CR for OSSM:
+### 1\. Grant Cluster-Admin Role (Version Specific)
+
+First, grant the Ingress Operator the cluster-admin role. This role is required in order for the Ingress Operator to create the SMCP CR for OSSM:
+
+  * **OpenShift 4.20 and beyond:** This step is no longer required and should be skipped.
+
+
 ```console
 $ oc adm policy add-cluster-role-to-user cluster-admin -z ingress-operator -n openshift-ingress-operator
 clusterrole.rbac.authorization.k8s.io/cluster-admin added: "ingress-operator"
 ```
-It is anticipated that this need for cluster-admin permissions will be eliminated in a future release of Gateway API in
-OpenShift.
+
+It is anticipated that this need for cluster-admin permissions will be eliminated in a future release of Gateway API in OpenShift.
+
+### 2\. Enable the Feature Gate
 
 Enable the feature gate for Gateway API. This will instruct the cluster Ingress Operator to install the Gateway API CRDs.
-Please note that, since feature gates are stored in the machine config, patching the feature gate config will result in
-the machine config operator rebooting each node sequentially, according to your machine config pool configuration:
+Please note that, since feature gates are stored in the machine config, patching the feature gate config will result in the machine config operator rebooting each node sequentially, according to your machine config pool configuration:
+* **OpenShift 4.20 and beyond:** This step is no longer required and should be skipped.
+
 ```console
 $ oc patch featuregates/cluster --type=merge --patch='{"spec":{"featureSet":"CustomNoUpgrade","customNoUpgrade":{"enabled":["GatewayAPI"]}}}'
 featuregate.config.openshift.io/cluster patched
 ```
 
 Wait for the ingress operator to successfully install the Gateway API CRDs:
+
 ```console
 $ oc get crd gatewayclasses.gateway.networking.k8s.io httproutes.gateway.networking.k8s.io gateways.gateway.networking.k8s.io referencegrants.gateway.networking.k8s.io
-NAME                                       CREATED AT
-gatewayclasses.gateway.networking.k8s.io   2023-04-04T18:02:55
-httproutes.gateway.networking.k8s.io        2023-04-04T18:02:55Z
-gateways.gateway.networking.k8s.io          2023-04-04T18:02:55Z
-referencegrants.gateway.networking.k8s.io   2023-04-04T18:02:56Z
+NAME                                        CREATED AT
+gatewayclasses.gateway.networking.k8s.io    2025-02-04T10:49:54Z
+httproutes.gateway.networking.k8s.io        2025-02-04T10:49:55Z
+gateways.gateway.networking.k8s.io          2025-02-04T10:49:54Z
```

Comment:
This whole section should be re-written with a Header like: "For OCP version before 4.20".